west.yml

```diff
@@ -73,7 +73,7 @@ manifest:
       groups:
         - hal
     - name: hal_infineon
-      revision: 5b0c7700a7be049e02835dfa0950d378aac39e67
+      revision: pull/6/head
```

Comment:
Should this be a sha1 instead?

Reply:
was updated in https://github.com/zephyrproject-rtos/zephyr/pull/44211/commits/6ba2ac774e528c672ba4ee3126090efbeb7c3b1c

Reply:
I see that. But it will make `git bisect` break in the future with this dangling reference (`pull/6/head`).

Reply:
rebased.